### ISSUE TITLE
Create UX design for data preview - single entry

### ISSUE THREAD

**Issue by igarashitm** [atlasmap/atlasmap#455]
We'd like to add data preview mode in AtlasMap UI. This ticket focuses on having a data preview only for the active mapping.

**Comment by paoloantinori** [comment]
@dongniwang I have talked with @igarashitm and the only thing that prevents us from entirely sign off the design are complications on the error messages, see https://github.com/atlasmap/atlasmap/issues/484

Do we have a chance to extract that feature from the current design or mark it as optional/deferred for a future enahncmeent?

cc: @amysueg

**Comment by kcbabo** [comment]
LGTM!  @gaughan can provide PM ack once he has a chance to review.

**Comment by gaughan** [comment]
LGTM also!

**Comment by dongniwang** [comment]
I know @igarashitm and @pleacu is on PTO right now, would it be possible for you @paoloantinori to sign off on the design? 

cc: @amysueg @heiko-braun

**Comment by gashcrumb** [comment]
FYI @paoloantinori I had a look and I don't see anything insurmountable, if there's any specifics we can deal with those during implementation, so I'll give a +1 from engineering too.

**Comment by paoloantinori** [comment]
Thanks @gashcrumb 

I have seen the new error message requirement. I assume we can match the request or in the wrong case they will be what we might not fully comply to, but I believe we are okay to go on.

**Comment by dongniwang** [comment]
Thank you @gashcrumb and @paoloantinori. I have put in a note in the design that error message is for future enhancement (not for current release) and is blocked by #484. 

Also, the UX design is available via [UX tracker.](https://syndesisio.github.io/syndesis-ux/) 

Closing this issue now. Thank you.
